Makefile

```diff
@@ -7,3 +7,46 @@ build:
 	rm -rf dist/__mocks__
 	rm -rf dist/setupTest.js
 	node build-scss.js
+
+transifex_langs = "ar,fr,es_419,zh_CN"
+i18n = ./src/i18n
+transifex_input = $(i18n)/transifex_input.json
+
+NPM_TESTS=build i18n_extract lint test is-es5
```

Comment:
I think the `is-es5` script can be removed, as Paragon does not have it defined?

Reply:
true, thanks!